client/state/sites/selectors.js

```diff
@@ -63,18 +62,40 @@ export const getSite = createSelector(
 
 		return {
 			...site,
-			...getComputedAttributes( site ),
 			...getJetpackComputedAttributes( state, siteId ),
 			hasConflict: isSiteConflicting( state, siteId ),
 			title: getSiteTitle( state, siteId ),
 			slug: getSiteSlug( state, siteId ),
 			domain: getSiteDomain( state, siteId ),
-			is_previewable: isSitePreviewable( state, siteId )
+			is_previewable: isSitePreviewable( state, siteId ),
+			options: getSiteOptions( state, siteId ),
 		};
 	},
 	( state ) => state.sites.items
 );
 
+export function getSiteOptions( state, siteId ) {
```

Comment:
Mmm, I actually don't think we should make this an exportable selector. It should only be used in `getSite`. Also probably the name of the function should be changed to reflect we are computing values and not just returning `site.options`.